world-if.css

```diff
@@ -0,0 +1,196 @@
+/* HEADER */
+.world-if-ArticleTemplate--imagecontainer-inner {
+  position: relative;
+}
+
+.world-if-ArticleTemplate--header {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  padding: 0;
+  background-color: rgba(0,0,0,.6);
+}
+
+.world-if-ArticleTemplate--section-section {
+  display: inline-block;
+  font-family: Neutra2Display-Medium;
+  color: #ffffff;
+  font-size: 22px;
+  line-height: 32px;
+  text-transform: uppercase;
+  background: #e3120b;
+  padding: 0 14px;
+  margin:0 7.5%;
+}
+
+.world-if-ArticleTemplate--flytitle {
+  display: inline-block; /* Overrides WIN template */
+  font-family: Neutra2Display-Medium;
+  color: #e3120b;
+  text-transform: uppercase;
+  font-size: 50px;
+  line-height: 44px;
+  margin: 10px 7.5% 0;
+}
+
+.world-if-ArticleTemplate--title {
+  font-family: Neutra2Display-Titling;
+  color: #ffffff;
+  text-transform: uppercase;
+  font-size: 50px;
+  line-height: 44px;
+  margin: 0 7.5% 15px;
+}
+
+@media screen and (max-width: 790px) {
+  .world-if-ArticleTemplate--section-section {
+    font-size: 18px; /* 18px */
+    line-height: 28px;
+    padding: 0 12px;
+  }
+  .world-if-ArticleTemplate--flytitle {
+    font-size: 36px; /* 35px & 30px */
+    line-height: 30px; /* 30px & 25px */
+    margin: 8px 7.5% 0;
+  }
+  .world-if-ArticleTemplate--title {
+    font-size: 36px; /* 35px & 30px */
+    line-height: 30px; /* 30px & 25px */
+    margin: 0 7.5% 10px;
+  }
+}
+
+@media screen and (max-width: 560px) {
+  .ArticleTemplate--header {
+    position: static; /* max-width 480px */
+    background: none;
+  }
+  .world-if-ArticleTemplate--section-section {
+    font-size: 14px; /* 1pc */
+    line-height: 22px;
+    padding: 0 10px;
+  }
+  .world-if-ArticleTemplate--flytitle {
+    font-size: 30px; /* 26px */
+    line-height: 26px; /* 23px */
+    margin: 8px 7.5% 0;
+  }
+  .world-if-ArticleTemplate--title {
+    font-size: 30px; /* 26px */
+    line-height: 26px; /* 23px */
+    color: #313131;
+    margin: 0 7.5%;
+  }
+}
+
+
+/* SUBHEADER */
+.world-if-ArticleTemplate--subheader {
+  width: 57.9%;
+  margin:0 7.5%;
+
+  border-left: 0; /* Override WIN template */
+  padding-left: 0; /* Override WIN template */
+  padding-top: 0; /* Override WIN template */
+}
+@media screen and (max-width: 860px) {
+  .world-if-ArticleTemplate--subheader {
+    width: 83.1%;
+    margin:0 7.5%;
+  }
+}
+.world-if-ArticleTemplate--pubdate {
```

Comment:
Mixed standard, empty lines between rules against no-lines. NTH a single standard in the file.